operator/controllers/provision.go

```diff
@@ -67,10 +68,11 @@ func (r *FlowTestReconciler) provisionResource(ctx context.Context) error {
 		Spec: v1.PodSpec{
 			Containers: []v1.Container{{
 				// TODO: Handle more than or less than 1 Container (#12)
-				Name:         referencePod.Spec.Containers[0].Name,
-				Image:        "k3d-rancher-logging-explorer-registry:5000/rancher-logging-explorer/pod-simulator:latest",
-				Args:         []string{"-log-dir", "/simulation.log"},
-				VolumeMounts: []v1.VolumeMount{{Name: "config-volume", MountPath: "/simulation.log", SubPath: "simulation.log"}},
+				Name:            referencePod.Spec.Containers[0].Name,
+				ImagePullPolicy: v1.PullIfNotPresent,
+				Image:           "mrsupiri/pod-simulator:latest",
```

Comment:
make this configurable as a flag in main.go

Reply:
Image name or the ImagePullPolicy? although getting references for those here will be diffcult

Reply:
image name